src/map.rs

```diff
@@ -48,6 +48,10 @@ pub struct Map {
     /// individual tiles may have different sizes. As such, there is no guarantee that this value
     /// will be the same as the one from the tilesets the map is using.
     pub tile_height: u32,
+    /// The stagger axis of Hexagonal tiled map.
+    pub staggeraxis: StaggerAxis,
+    /// The stagger index of Hexagonal tiled map.
+    pub staggerindex: StaggerIndex,
```

Comment:
I think the naming style should be `stagger_axis` and `stagger_index`.

Also please just say "map" instead of "tiled map", mainly for consistency with the rest of the documented fields.

Also, this applies not only to `Hexagonal` but also to `Staggered` (isometric tiles) orientation.